<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>EnsembleViewBase</class>
 <widget class="QWidget" name="EnsembleViewBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ensemble View</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="mTitleLbl" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth" >
      <number>3</number>
     </property>
     <property name="text" >
      <string>Cluster Management</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QListView" name="mClusterListView" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mRefreshBtn" >
           <property name="text" >
            <string>&amp;Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mAddBtn" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Add a cluster node.&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="whatsThis" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Add a cluster node.&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mRemoveBtn" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Remove selected cluster node.&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="whatsThis" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Remove selected cluster node.&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="mNodeGroup" >
         <property name="title" >
          <string>Node Settings</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QLabel" name="mHostnameLbl" >
            <property name="text" >
             <string>Hostname:</string>
            </property>
            <property name="buddy" >
             <cstring>mHostnameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="mIpAddressEdit" >
            <property name="autoFillBackground" >
             <bool>false</bool>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="mNameLbl" >
            <property name="text" >
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="mHostnameEdit" >
            <property name="readOnly" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="mCurrentOsEdit" >
            <property name="autoFillBackground" >
             <bool>true</bool>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="mIpAddressLbl" >
            <property name="text" >
             <string>IP Address:</string>
            </property>
            <property name="buddy" >
             <cstring>mIpAddressEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="mNameEdit" />
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="mCurrentOsLbl" >
            <property name="text" >
             <string>Current OS:</string>
            </property>
            <property name="buddy" >
             <cstring>mCurrentOsEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="mOtherLbl" >
            <property name="text" >
             <string>Other:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QTableView" name="mSettingsTableView" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>mClusterListView</tabstop>
  <tabstop>mRefreshBtn</tabstop>
  <tabstop>mAddBtn</tabstop>
  <tabstop>mRemoveBtn</tabstop>
  <tabstop>mNameEdit</tabstop>
  <tabstop>mHostnameEdit</tabstop>
  <tabstop>mIpAddressEdit</tabstop>
  <tabstop>mCurrentOsEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
